migrations/versions/4fedf700f147_.py

```diff
@@ -0,0 +1,30 @@
+"""Added Parties to system. Many parties to one fraternity. In addition, parties have a name and date column.
+
+Revision ID: 4fedf700f147
+Revises: cf49f49a2c66
+Create Date: 2016-04-09 11:02:28.450687
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '4fedf700f147'
+down_revision = 'cf49f49a2c66'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('fraternities', sa.Column('date', sa.Date(), nullable=False))
+    op.add_column('fraternities', sa.Column('fraternity_id', sa.Integer(), nullable=False))
+    op.create_foreign_key(None, 'fraternities', 'fraternities', ['fraternity_id'], ['id'])
```

Comment:
It looks like you're trying to add "parties", but you added "fraternities" instead.